cspell.json

```diff
@@ -15,7 +15,8 @@
     "Suspendable",
     "Failable",
     "livestreams",
-    "livestream"
+    "livestream",
+    "unsubscribable"
```

Comment:
I can't see where this is used in the PR, is it still needed?

Reply:
I've removed as I don't think it was needed.

Reply:
i forgot it, thanks!